monitor.js

```diff
@@ -2,15 +2,24 @@
  * Monitor
  * Pokemon Showdown - http://pokemonshowdown.com/
  *
- * Various utility functions to make sure PS is running healthy.
+ * Various utility functions to make sure PS is running healthily.
  *
  * @license MIT license
  */
 'use strict';
 
 const FS = require('./fs');
 
+/**
+ * @class TimedCounter
```

Comment:
I feel like it looks inconsistent to not include when classes that need `@property` documentation include it, but looking at the documentation for the Typescript you've written, you don't use `@property` and use `@type` for each property in the constructor instead. What should the style be for documenting classes?